.github/workflows/test_suite_linux.yml

```diff
@@ -112,6 +112,7 @@ jobs:
       run: |
         export LIBGL_ALWAYS_SOFTWARE=true
         export WEBOTS_DISABLE_SAVE_SCREEN_PERSPECTIVE_ON_CLOSE=true
+        xvfb-run --auto-servernum python scripts/packaging/update_worlds_and_protos.py ${{ github.sha }}
```

Comment:
Should we do this for macOS and Windows? Also, should we target a tag for the release?

Reply:
For the release, it is done in https://github.com/cyberbotics/webots/blob/0bbd6fa4b003062b76e123e29a9d945e49d374c4/scripts/packaging/Makefile#L70-L73. For the tests, they are currently not implemented for macOS and Windows.